### PR TITLE
Fixes 01

### PR DESCRIPTION
**Pull request by Inego**

### CODE REVIEW

**Comment by soywiz** [comment]
@Inego is this PR ready for reviewing?

**Comment by Inego** [comment]
I wanted to add a commit here with removal of `View.propagateEvents` but didn't, because there are unanswered questions in Discord.

**Comment by Inego** [comment]
OK, now it's ready for review!

**Comment by Inego** [comment]
Sorry for force-pushes :(

File: korge/src/commonMain/kotlin/com/soywiz/korge/view/View.kt [hunk 1]
```diff
@@ -1468,12 +1466,10 @@ fun <T : View> T.onNextFrame(updatable: T.(views: Views) -> Unit) {
...
-        node = node.parent
-        if (node != null) {
-            count++
-        }
+    var parent = this?.parent
+    while (parent != null) {
+        count++
+        parent = parent.parent
     }
     return count
```
Comment (soywiz) [inline korge/src/commonMain/kotlin/com/soywiz/korge/view/View.kt:1474]: Risky, but I think this is correct. Also there are tests covering this, so I guess this is ok

File: samples/spriteanim/src/commonMain/kotlin/Main.kt [hunk 2]
```diff
@@ -49,50 +51,100 @@ suspend fun main() = Korge(width = 512, height = 512) {
...
+    addChild(player1)
+    addChild(player2)
+
+    addUpdater { time ->
+        val scale = 16.milliseconds / time
+        val disp = 2 * scale
+        val keys = views.input.keys
+
+        player1.handleKeys(keys, disp)
+        player2.handleKeys(keys, disp)
```
Comment (soywiz) [inline samples/spriteanim/src/commonMain/kotlin/Main.kt:126]: Cool

**Review by soywiz — APPROVED** [review]

**Pull request merged**